pkg/devicetwin/dttype/types_test.go

```diff
@@ -0,0 +1,630 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dttype
+
+import (
+	"encoding/json"
+	"errors"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/satori/go.uuid"
+)
+
+// Test_SetEventID is function to test SetEventID().
+func Test_SetEventID(t *testing.T) {
+	tests := []struct {
+		name      string
+		timestamp int64
```

Comment:
SetEventID does not require timestamp to be set right ?

Reply:
Updated